extensions/stackdriver/metric/registry.cc

```diff
@@ -42,6 +44,20 @@ StackdriverOptions getStackdriverOptions(
                                        grpc::InsecureChannelCredentials());
     options.metric_service_stub =
         google::monitoring::v3::MetricService::NewStub(channel);
+  } else if (!sts_port.empty()) {
+    ::grpc::experimental::StsCredentialsOptions sts_options;
+    sts_options.token_exchange_service_uri =
+        "http://127.0.0.1:" + sts_port + "/token";
+    sts_options.subject_token_path = "/var/run/secrets/tokens/istio-token";
+    sts_options.subject_token_type = "urn:ietf:params:oauth:token-type:jwt";
+    sts_options.scope = "https://www.googleapis.com/auth/cloud-platform";
```

Comment:
since these values are used in two places and not obvious, either make constants or consolidate the two blocks in one file.

Reply:
Moved constant strings to one place.